code/game/machinery/pipe/pipe_construction.dm

```diff
@@ -540,7 +540,7 @@
 /obj/item/pipe_meter/wrench_act(mob/living/user, obj/item/I)
 	if(!locate(/obj/machinery/atmospherics/pipe, loc))
 		to_chat(user, "<span class='warning'>You need to fasten it to a pipe.</span>")
-		return TRUE
+		return
```

Comment:
Should this return `FALSE`?

Reply:
Uhh, I am not entirely sure. It appears to work as is for both the RPD and BRPD. https://i.imgur.com/33le5ii.png